specification/compute/resource-manager/Microsoft.Compute/GalleryRP/stable/2024-03-03/gallery.json

```diff
@@ -1737,6 +1737,518 @@
+              "$ref": "../../../common-types/v1/common.json#/definitions/CloudError"
+            }
+          }
+        },
+        "x-ms-long-running-operation": true,
+        "x-ms-examples": {
+          "Update a gallery inVMAccessControlProfile.": {
+            "$ref": "./examples/galleryResourceProfileExamples/GalleryInVMAccessControlProfile_Update.json"
+          }
+        }
+      },
+      "get": {
+        "tags": [
+          "GalleryInVMAccessControlProfiles"
+        ],
+        "operationId": "GalleryInVMAccessControlProfiles_Get",
+        "description": "Retrieves information about a gallery inVMAccessControlProfile.",
+        "parameters": [
+          {
+            "$ref": "../../../common-types/v1/common.json#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "name": "galleryName",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "The name of the Shared Image Gallery from which the InVMAccessControlProfiles are to be retrieved."
+          },
+          {
+            "name": "inVMAccessControlProfileName",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "The name of the gallery inVMAccessControlProfile to be retrieved."
+          },
+          {
+            "$ref": "../../../common-types/v1/common.json#/parameters/ApiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK",
+            "schema": {
+              "$ref": "#/definitions/GalleryInVMAccessControlProfile"
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "../../../common-types/v1/common.json#/definitions/CloudError"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "Get a gallery inVMAccessControlProfile.": {
+            "$ref": "./examples/galleryResourceProfileExamples/GalleryInVMAccessControlProfile_Get.json"
+          }
+        }
+      },
+      "delete": {
+        "tags": [
+          "GalleryInVMAccessControlProfiles"
+        ],
+        "operationId": "GalleryInVMAccessControlProfiles_Delete",
+        "description": "Delete a gallery inVMAccessControlProfile.",
+        "parameters": [
+          {
+            "$ref": "../../../common-types/v1/common.json#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "name": "galleryName",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "The name of the Shared Image Gallery in which the InVMAccessControlProfile it to be deleted."
```

Comment:
The name of the Shared Image Gallery in which the InVMAccessControlProfile resides.

Reply:
updated